app/dts/behaviors/key_tap.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2025 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/behaviors.h>
+
+/ {
+    macros {
+#if ZMK_BEHAVIOR_OMIT(KTAP)
+    /omit-if-no-ref/
+#endif
+        ktap: key_tap {
+            compatible = "zmk,behavior-macro-one-param";
+            #binding-cells = <1>;
+            display-name = "Key Tap";
+            bindings = <&macro_param_1to1>, <&kp MACRO_PLACEHOLDER>;
```

Comment:
```suggestion
            bindings = <&macro_param_1to1>, <&kp MACRO_PLACEHOLDER>;
            tap-ms = <30>;
```
Might be worth explicitly setting this (the default from `ZMK_MACRO_DEFAULT_TAP_MS` is 30) so that it doesn't necessarily modify a pre-defined behavior?